### ISSUE TITLE
Automate re-generation of README.md and metadata.yaml in PRs

### ISSUE THREAD

**Issue by NimJay** [GoogleCloudPlatform/cloud-foundation-toolkit#2470]
### Component

```markdown
Pull-request automation
```

### TL;DR

It would be convenient if there was some GitHub bot that could run `make docker_generate_docs` for us and git-commit updates to the README.md and metadata.yaml files — on the source branches of pull-requests.

### Detailed design

Example user experience: I add a comment like  "@some-bot-name" in the pull-request discussion thread. The bot then (in its own environment) runs `make docker_generate_docs` and makes commits to the source branch of the PR to update the metadata.yaml and/or README.md file(s).

Alternatively, could we run `make docker_generate_docs` and output the result `README.md` and `metadata.yaml` in the output of the `lint` GitHub Action? This means we could merely copy-and-paste from the `lint` GitHub Action log output.

### Additional information

My feature request might be over-engineering. So I'm indifferent about the exact solution. I would like to avoid having to run `make docker_generate_docs` often. (Also, this is not a high priority.)
